docs/system_overview/federation.md

```diff
@@ -161,3 +161,65 @@ In order to support commands such as `ils` and `ilsresc` across a 3.x to 4.0+ fe
 
 The effect of turning this negotiation off is a lack of SSL encryption when talking with a 3.x zone.  All clients that connect to this 4.0+ zone will also need to disable the Advanced Negotiation in their own 'irods_environment.json' files.
 
+### How to enable federation between 3.x and 4.x:
+
+The steps required for enabling federation between a 3.x zone and a 4.x zone will be described through an example.
+
+Let ZoneA have the following properties:
```

Comment:
A is 4.x and B is 3.x?  please make that clearer up top.

Reply:
I added explicit mentions of the versions for the two zones on these lines. Thanks!